WLogger/WeekLogs/WeekLogsViewModel.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  WeekLogsViewModel.swift
+//  WLogger
+//
+//  Created by Pablo Giorgi on 1/27/16.
+//  Copyright © 2016 Wolox. All rights reserved.
+//
+
+import Foundation
+import ReactiveCocoa
+
+public class WeekLogsViewModel {
+
+    public let weekTotalTitle: String
+    
+    private let logsAmount = MutableProperty<String>("")
+    public let fetchLogsAmount: Action<AnyObject, String, RepositoryError>
+    
+    private let loggerImage = MutableProperty<UIImage?>(nil)
+    public let fetchLoggerImage: Action<AnyObject, UIImage?, NSError>
+    
+    public let weekLogsTableViewModel: WeekLogsTableViewModel
+    
+    public init(imageFetcher: ImageFetcher, logRepository: LogRepository) {
+        weekTotalTitle = "week total"
+        weekLogsTableViewModel = WeekLogsTableViewModel(logRepository: logRepository)
+
+        fetchLogsAmount = Action { _ in
```

Comment:
I think these two actions should not be ```Action```s, since they are not triggered from a controller action. 
Instead it may be just observable properties. What do you think?